### DISCUSSION TITLE
niel$ command alternative for zash

### DISCUSSION THREAD

**Discussion by nevilrana** [mrdbourke/zero-to-mastery-ml#50] [General]
I have setup the anaconda as you explianed for mac, but i have setup it in zash so now when i am trying to activate conda or launch juypter notebook it is showing command not found error (niel$)

**Comment by mrdbourke** [comment]
Hi @nevilrana , 

What have you tried so far? 

Are you able to post your errors/code you're trying to run? 

Perhaps that would be a better help! 

Did you try to activate your environment when you went into the terminal the next time? (the environment has to be active whenever you try to launch a jupyter notebook)

**Comment by Erick-jah** [comment]
Good morning, guys!
I am new here, and I just enrolled in the program yesterday. 
I am glad to be part of the program!

**Comment by Erick-jah** [comment]
My name is Erick Jah, by the way.
